spec/controllers/presentations_controller_spec.rb

```diff
@@ -164,6 +198,18 @@
         end
       end
     end
+
+    describe 'PATCH #update' do
```

Comment:
Cool, that makes sense to me! I spent a while trying to rearrange a lot of these methods to fit into the pattern we already had for the project, but I think it would make it much more organized to do it the way you described! I will refactor and update.